x-pack/plugins/ingest_manager/common/types/models/agent.ts

```diff
@@ -62,7 +62,7 @@ export interface AgentEvent extends NewAgentEvent {
   id: string;
 }
 
-export interface AgentEventSOAttributes extends NewAgentEvent, SavedObjectAttributes {}
+export type AgentEventSOAttributes = NewAgentEvent;
```

Comment:
removed `SavedObjectAttributes` since it requires `AgentEventSOAttributes` to define index signature